ipa-core/src/test_fixture/ipa.rs

```diff
@@ -202,8 +202,17 @@ pub async fn test_oprf_ipa<F>(
     };
 
     let aws = config.attribution_window_seconds;
-    let result: Vec<_> = world
-        .semi_honest(
+    let result: Vec<_> = if config.per_user_credit_cap == 256 {
+        world.semi_honest(
+            records.into_iter(),
+            |ctx, input_rows: Vec<OPRFIPAInputRow<BA5, BA8, BA20>>| async move {
+                oprf_ipa::<BA5, BA8, BA16, BA20, BA8, 32>(ctx, input_rows, aws)
+                    .await
+                    .unwrap()
+            },
+        )
+    } else {
+        world.semi_honest(
```

Comment:
Why is this case, of 256, different from the others? There's already a big match statement below, why couldn't this case just be added there?

Reply:
On the other side of the conditional, the credit cap is the only parameter that changes. This one case has a bunch of parameters that differ. I will add a comment noting this.

The purpose of this change was to support collect_steps coverage. Between two possible upcoming changes (new compact gate and runtime parameter selection), it didn't seem important to put a lot of effort into structuring this optimally right now.